.github/workflows/dockerimage.yml

```diff
@@ -8,15 +9,17 @@ jobs:
         image:
         - v2-apache
         - v3-nginx
+
     steps:
     - uses: actions/checkout@v1
       with:
         fetch-depth: 1
 
-    - name: Log in to Docker Hub
-      run: docker login
-           -u ${{ secrets.dockerhub_user }}
-           -p ${{ secrets.dockerhub_token }}
+    - uses: azure/docker-login@v1
+      if: github.event_name == 'push' && github.ref == 'refs/heads/master'
```

Comment:
The event name is probably unneeded. Note the `on: push` on line 2 above. Do you see any issue with that?

Reply:
You are exactly right! (bad copy and paste)